appveyor.yml

```diff
@@ -10,11 +10,11 @@ init:
   - cmd: set PATH=%PYTHON%;%PYTHON%\scripts;%PATH%
   - cmd: python --version
   - cmd: pip --version
-  - cmd: pip install cx_Freeze nose pycodestyle python-docx 
+  - cmd: pip install cx_Freeze coverage pycodestyle pytest pytest-cov pytest-testdox python-docx 
 
 test_script:
   - cmd: pycodestyle . --exclude=backend/data_xml.py
-  - cmd: nosetests tests.py
+  - cmd: py.test --cov
```

Comment:
it should limit coverage report to `frontend` and `backend` module.
```
py.test --cov=frontend --cov=backend
```

Reply:
We need also to upload coverage report to codecov.io.

To do so, we need to install `codecov` using `pip`, reformat coverage report to xml (`coverage xml`) and push it to codecov (`codecov --file coverage.xml`).

Reply:
Plus remember to add codecov badge to README